packages/create-platformatic/src/index.mjs

```diff
@@ -30,6 +30,14 @@ const createPlatformatic = async (argv) => {
     await say(`${greeting} welcome to ${version ? `Platformatic ${version}!` : 'Platformatic!'}`)
     await say('Let\'s start by creating a new project.')
 
+    const currentVersion = process.versions.node
+    const supported = isCurrentVersionSupported(currentVersion)
+    if (!supported) {
+      const supportedVersions = getSupportedNodeVersions().join(', ')
+      console.warn(`Node.js v${currentVersion} is out of date and unsupported!`)
```

Comment:
@simonplend can you please check these message? Maybe we shouldn't say that the version is "out of date".

Reply:
@marcopiraccini should we use the `say` method instead?

Reply:
@mcollina Yes, good idea. Also, shall we exit in these cases?

Reply:
Would the following be a better warning message? 
```
Platformatic is not supported on Node.js v${currentVersion}.
Please use one of the following Node.js versions >= v16.17.0 or >= 18.8.0 or >= 19.0.0.
```

Reply:
I know I'm late to the party as this has already been merged, but yes, I think that would be a better warning message as it's much more explicit.

Reply:
@simonplend would you like me to open up a new PR and link it to original issue with just the above message changes?

Reply:
do that!